constants.py

```diff
@@ -1,2 +1,9 @@
 
-drivetrain_turn_gear_ratio = 80.4848
\ No newline at end of file
+drivetrain_turn_gear_ratio = 80.4848
+robot_length = 29
```

Comment:
Perhaps an application for that fancy unit library?

Reply:
I'm trying to move away from Unum because it seemed to slow down the robot significantly last year. We could do something like "inch: float" somewhere for typing purposes. @mllewellyn

Reply:
Interesting about the unum performance issues. I didn't see that one coming.  I guess that library is heaver weight than I thought.

`inch: float` seems like a reasonable solution. You could also include it in the variable name eg `robot_length_in` or even just in a comment. Anything works for me as long as it's written down in the code.